internal/parser/parser/office_table_render_test.go

```
@@ -0,0 +1,286 @@
+		t.Fatalf("want 2 <table> chunks, got %d", n)
+	}
+	if n := strings.Count(out, "<tr><th>C1</th><th>C2</th></tr>"); n != 2 {
+		t.Fatalf("want header repeated in both chunks, got %d", n)
+	}
+}
+
+// TestXLSXParser_HeaderAndCaption asserts the XLSX parser emits a <caption> and
+// renders the first row as <th>, and that the header text appears only in <th>.
+func TestXLSXParser_HeaderAndCaption(t *testing.T) {
+	data := newTestXLSX(t, func(f *excelize.File) {
+		f.SetCellValue("Sheet1", "A1", "Product")
```

golangci-lint (2.12.2)

[error] Error return value of `f.SetCellValue` is not checked

(errcheck)